manifests/reposync-crd.yaml

```diff
@@ -1187,30 +1187,17 @@ spec:
                   values:
                     description: values to use instead of default values that accompany
                       the chart. Format values the same as default values.yaml. These
-                      values will take precedence if valuesFileRefs is also specified.
-                      How to handle multiple valuesFiles is determined by `valuesFileApplyStrategy`.
+                      values will take precedence if `valuesFileRefs` is also specified.
```

Comment:
Suggest:
If valuesFileRefs is also specified, fields from values will override fields from valuesFileRefs.

Reply:
done